src/components/modal/personalization/utils.ts

```diff
@@ -28,8 +30,24 @@ export function initializePersonalizationConfig(metaInfo: MetaInfo[]) {
     return [languages, technologies, products];
 }
 
-// TODO: you will also need to pass userId here to construct the path for the PUT
-// eslint-disable-next-line
-export function submitPersonalizationSelections(body: any) {
-    // do the http req here
+export async function submitPersonalizationSelections(
+    {
+        followedTags,
+        emailPreference,
+    }: {
+        followedTags: Array<Tag>;
+        emailPreference: boolean;
+    },
+    userId: unknown
```

Comment:
I think we can change this from unknown if we add `userId` [here](https://github.com/mongodb/devcenter/blob/main/next-auth.d.ts#L8).